src/DGmethods/DGmodel.jl

```diff
@@ -6,126 +6,112 @@ struct DGModel{BL,G,NFND,NFD,GNF}
   gradnumflux::GNF
 end
 
-function (dg::DGModel)(dQdt, Q, param, t; increment=false)
+function (dg::DGModel)(dYdt, Y, param, t; increment=false)
```

Comment:
Quoting Tapio:

> > Q/q = state variables
> 
> `Q` is a diabatic source, `q` a specific humidity in other parts of the code. Please avoid them for states. We choose `Y` for state in the design document. In the code, something more verbose would be better (`state` etc., as in the atmosphere model)

I think `u` is a bad choice as it is also the component of velocity in the 'x'-direction. Andre and I used it when learning DG and it led to some confusion because of that

Reply:
That seems reasonable.